src/react-apps/applications/runtime/src/utils/validation.ts

```diff
@@ -91,8 +91,7 @@ export function validateFormComponents(
   formLayout.forEach((component: any) => {
     if (!component.hidden) {
       if (component.type === 'FileUpload') {
-        if (component.minNumberOfAttachments > 0 && numberOfAttachments < 1 ||
```

Comment:
Sånn som jeg forstår det så er `attachments`  et objekt med vedleggs-komponent-id som nøkler, og for hver vedleggskomponent (nøkkel) er det et array med vedlegg som hører til den komponenten. Derfor virker ikke `numberOfAttachments` å representere noe vi kan bruke her. @lorang92 Rett gjerne på meg hvis jeg har misforstått noe her.

Reply:
Helt enig :)